qemu/tests/balloon_hotplug.py

```diff
@@ -37,6 +39,19 @@ def run_pm_test(pm_test, plug_type):
                               % (pm_test, plug_type), logging.info)
         utils_test.run_virt_sub_test(test, params, env, pm_test)
 
+    def enable_balloon_service(session):
+        """
+        Install balloon service and check its status in windows guests
+        :param session: shell Object
+        """
+        if params['os_type'] == 'windows':
```

Comment:
No need to judge os_type, this function only works on windows guest .
From line 85
blnsrv_enable has determined this function run in windows guest.

Reply:
Yes, I knew that.
I add this line just for a double ensure.
Just in case someone use this param in mistakely in future.

Reply:
I think you have noted it clearly, in docstring and cfg file

Reply:
It's for better extendibility, this function maybe called somewhere else not specific for blnsrv_enable flag.